easybuild/easyconfigs/j/jax/jax-0.2.18-correct_libraries.patch

```diff
@@ -0,0 +1,82 @@
+# using system environment variables + reparing build.py script + using EB downloaded TensorFlow tarball
+diff -ruN jax_orig/build/build.py jax/build/build.py
+--- jax_orig/build/build.py	2021-07-30 10:18:43.599233082 +0200
++++ jax/build/build.py	2021-08-05 17:28:04.071995000 +0200
+@@ -208,15 +208,20 @@
+ 
+ 
+ def check_bazel_version(bazel_path):
+-  try:
+-    version_output = shell([bazel_path, "--bazelrc=/dev/null", "version"])
+-  except subprocess.CalledProcessError:
+-    return False
+-  match = re.search("Build label: *([0-9\\.]+)[^0-9\\.]", version_output)
+-  if match is None:
+-    return False
+-  actual_ints = [int(x) for x in match.group(1).split(".")]
+-  return actual_ints >= [2, 0, 0]
++    # try:
```

Comment:
Why is this commented? the commented code is actually correct

Reply:
Yeah with that you need to use `actual_ints = [int(x) for x in match.group(2).split(".")]` as you introduced a new group
However what about simply using the 2 commits/changes I linked above?

Reply:
Or better: Use jaxlib 0.1.70 which already includes that change and does not require this patch :)

Reply:
done, using 0.1.70